lib/CustomersApi.php

```diff
@@ -66,9 +66,10 @@ public function setApiClient($apiClient) {
    * @param int $offset How many results to skip. (optional)
    * @param string $search Search term. (optional)
    * @param string $status Customer status. (optional)
+   * @param string $email Customer email. (optional)
    * @return CustomerListResponse
    */
-   public function _list($limit = null, $offset = null, $search = null, $status = null, $headers = null) {
+   public function _list($limit = null, $offset = null, $search = null, $status = null, $email = null, $headers = null) {
```

Comment:
This might require a major new version since if you were passing headers before it will now think they're an email.

Reply:
Ah, yeah. I think that makes sense.

Reply:
I'm not sure I like the idea of doing a major version bump just for added a new param. Is there an alternative solution to where this can be added in a backwards compatible way, or prevent us in the future from performing another major version bump the next time this comes up?

Reply:
Agreed. Here's a follow up commit. Give it look when you get a chance! - https://github.com/Dwolla/dwolla-swagger-php/pull/47/commits/6b1480c724a033826d23044ab6fad1d55b8aadbe